### DISCUSSION TITLE
Global Image/Media Directory

### DISCUSSION THREAD

**Discussion by Gomilu** [GitbookIO/community#196] [Feature Requests]
### What's your feature/idea?

I would like to access my files, images, drawings in all different spaces

### What's your desired outcome?

 I could go to Space A, insert a drawing/Image I did within Space B and vice versa

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by taszap** [comment]
Closing in favour of #234
